DepositApi/Controllers/CalculateDepositApiController.cs

```diff
@@ -13,35 +13,37 @@ namespace DepositApi.Controllers
 {
     [ApiController]
     [Route("/api/calculate")]
-    public class DepositCalcApiController : ControllerBase
+    public class DepositCalculationApiController : ControllerBase
     {
         private readonly IDepositService depositService;
         private readonly IMapper mapper;
 
-        public DepositCalcApiController(IDepositService depositService, IMapper mapper)
+        public DepositCalculationApiController(IDepositService depositService, IMapper mapper)
         {
             this.depositService = depositService;
             this.mapper = mapper;
         }
 
         [HttpGet]
-        public ActionResult Get(DepositModel deposit)
+        public async Task<ActionResult> Get(DepositModel deposit)
```

Comment:
Please, use FromQuery attribute for the deposit parameter here

Reply:
Why do we need to use FromQuery attribute? I thought we should get parameters from the body because we send JSON to API.

Reply:
I added FromQuery attribute and changed requests in UI